sktime/forecasting/compose/_ensemble.py

```diff
@@ -61,16 +61,17 @@ class AutoEnsembleForecaster(_HeterogenousEnsembleForecaster):
     --------
     EnsembleForecaster
 
-    Example
-    -------
+    Examples
+    --------
     >>> from sktime.forecasting.compose import AutoEnsembleForecaster
     >>> from sktime.forecasting.naive import NaiveForecaster
     >>> from sktime.forecasting.trend import PolynomialTrendForecaster
     >>> from sktime.datasets import load_airline
     >>> y = load_airline()
-    >>> forecasters = [("trend", PolynomialTrendForecaster()),\
-                        ("naive", NaiveForecaster())]
-    >>> forecaster = AutoEnsembleForecaster(forecasters=forecasters, n_jobs=2)
+    >>> forecasters = [
+    ...     ("trend", PolynomialTrendForecaster()),
+    ...     ("naive", NaiveForecaster())]
```

Comment:
if you want this filly linted, the last `]` should be on a new line and line 73 should end in `,`